boefjes/tests/test_tasks.py

```diff
@@ -121,16 +121,16 @@ def test_handle_boefje_with_exception(self, mock_find_ooi_in_past, mock_bytes_ap
             BoefjeHandler(LocalBoefjeJobRunner(local_repository), local_repository).handle(meta)
 
         mock_bytes_api_client.save_boefje_meta.assert_called_once_with(meta)
-        mock_bytes_api_client.save_raw.assert_called_once_with(
-            "some-random-job-id",
-            "Boefje failed",
-            {
-                "error/boefje",
-                "dummy_boefje_runtime_exception",
-                "boefje/dummy_boefje_runtime_exception",
-                f"boefje/dummy_boefje_runtime_exception-{meta.parameterized_arguments_hash}",
-            },
-        )
+        mock_bytes_api_client.save_raw.assert_called_once()
+        assert mock_bytes_api_client.save_raw.call_args[0][0] == "some-random-job-id"
+        assert "Traceback (most recent call last)" in mock_bytes_api_client.save_raw.call_args[0][1]
+        assert "JobRuntimeError: Boefje failed" in mock_bytes_api_client.save_raw.call_args[0][1]
+        assert mock_bytes_api_client.save_raw.call_args[0][2] == {
+            "error/boefje",
+            "dummy_boefje_runtime_exception",
+            "boefje/dummy_boefje_runtime_exception",
+            f"boefje/dummy_boefje_runtime_exception-{meta.parameterized_arguments_hash}",
+        }
```

Comment:
Nice one. I think readability could be a bit improved by extracting `mock_bytes_api_client.save_raw.call_args` as a variable, e.g.:
```python3
call_cargs = mock_bytes_api_client.save_raw.call_args
```